incubator/openldap/templates/openldap-pvc.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# A PVC for persisting data
+#
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "openldap.fullname" . }}
+  labels:
+    app: {{ template "openldap.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  accessModes:
+    - {{ .Values.persistence.accessMode | quote }}
```

Comment:
I'd like you to make list indentation consistent throughout the chart. Either use zero or two spaces.